File: TvShowApp/app/src/main/java/com/gm/gislain/tvshowapp/Model/Genre.java
package com.gm.gislain.tvshowapp.Model;

/**
 * Created by Gislain on 25/11/2016.
 */

public class Genre {

    private int id;
    private String genre;

    public static final String KEY_ID = "id";
    public static final String KEY_GENRE = "genre";

    public Genre() {}

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public String getGenre() {
        return genre;
    }

    public void setGenre(String genre) {
        this.genre = genre;
    }
}

File: TvShowServer/Model/genre.py
class Genre(object):
    def __init__(self, id, genre):
        self.id = id
        self.genre = genre

    def getId(self):
        return self.id

    def getGenre(self):
        return self.genre

    def serialize(self):
        return {'id': self.id, 'genre': self.genre}
